### PR TITLE
users/v-jandrew/refresh-extensionssection

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**
AB#43217021

### CODE REVIEW

File: microsoft-edge/extensions-chromium/getting-started/extension-sideloading.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 11/07/2022
+ms.date: 02/03/2023
 ---
 # Sideload an extension
 
-During development, you can use the Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension.
+During development, you can use Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension without changing any settinga. This helps you test new extensions safely.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/extension-sideloading.md:12]:
```suggestion
During development, you can use Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension without changing any settings. This helps you test new extensions safely.
```

File: microsoft-edge/extensions-chromium/getting-started/index.md [hunk 2]
```diff
@@ -5,13 +5,13 @@ author: MSEdgeTeam
...
 ms.prod: microsoft-edge
-ms.date: 11/07/2022
+ms.date: 02/07/2022
 ---
 # Extension concepts and architecture
 
 This article introduces concepts about building a Microsoft Edge extension.  Follow along to understand how multi-tab browsers work.
 
-A Microsoft Edge *extension* is a small app that adds or modifies features of Microsoft Edge.  For an introduction, see [Overview of Microsoft Edge extensions](../index.md).
+A Microsoft Edge *extension* is a small app that users may opt into that adds or modifies features of Microsoft Edge for a unique user experience.  For an introduction, see [Overview of Microsoft Edge extensions](../index.md).
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/index.md:14]: probably reword "for a unique user experience"; not sure the objective or value-add is to have a "unique user experience".

To draft another wording, I'd have to read the positioning phrases at
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/
Here's an existing phrasing from that Intro page that's pretty focused:
"An extension provides a niche experience or function that is important to a target audience."

I'd try wording like "specialized" -- I'd read the overview article and pick wording from there.

Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/getting-started/index.md:14]:
```suggestion
A Microsoft Edge *extension* is a small app that users may opt into that adds or modifies features of Microsoft Edge to provide a specialized experience or function that is important to a target audience.  For an introduction, see [Overview of Microsoft Edge extensions](../index.md).
```

File: microsoft-edge/extensions-chromium/index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ To work directly with a specific part of the browser, such as a window or tab, y
 <!-- ====================================================================== -->
 ## Basic guidance
 
-Great places to begin your extension development tutorials and documentation research are sites hosted by the browser organizations.  Some of the most popular browsers to build extensions for include Safari, Firefox, Chrome, Opera, Brave, and Microsoft Edge. 
+Great places to begin your extension development tutorials and documentation research are sites hosted by the browser organizations.  Some of the popular browsers to build extensions for include Safari, Firefox, Chrome, Opera, Brave, and Microsoft Edge.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/index.md:30]:
```suggestion
Great places to begin your extension development tutorials and documentation research are sites hosted by the browser organizations.  Some of the popular browsers to build extensions for include Microsoft Edge, Safari, Firefox, Chrome, Opera, and Brave. 
```
listing Edge last seems to set up a mixed message, 
seems appropriate and clarifying/consistent to list it first

File: microsoft-edge/extensions-chromium/getting-started/index.md (hunk 2 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/extensions-chromium/getting-started/index.md:14]: sounds smart

Comment (JasonAndrewWriter) [reply microsoft-edge/extensions-chromium/getting-started/index.md:14]: So I added specialized to that phrase.  Later on there is the term unique identifier so it seems wise.

**Review by mikehoffms — APPROVED** [review]

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**